docs/users-guide/09-multi-series-charting.md

```diff
@@ -0,0 +1,85 @@
+06-sharing-answers.md
+
+# Charts with multiple series
+
+
+Data in isolation is rarely all that useful. One of the best ways to add context and clarity when communicating with data is to show data side-by-side with other data. Here are just a few  examples of data that is better together than apart.
+
+- Your company’s revenue vs. its costs over time
+- Average order price this month and user signups for that month
+- New users per day vs. returning users per day.
+- Orders per day per from a few different product lines.
+
+## In Metabase there are two main ways to get data side-by-side.
+
+1. Combining two existing saved questions that share a common dimension (like time) on a dashboard
+
+e.g. Let me see revenue over time and cost over time together.
+
+
+2. Asking a question that involves multiple dimensions in the query builder (or in SQL if you’re fancy).
```

Comment:
the numbering isn't coming through properly.  i think you need to remove some of those empty lines and condense this.